plugins/config-schema/src/components/SchemaView/SchemaView.tsx

```diff
@@ -15,12 +15,22 @@
  */
 
 import React from 'react';
-import { ArrayView } from './ArrayView';
-import { MatchView } from './MatchView';
-import { ObjectView } from './ObjectView';
-import { ScalarView } from './ScalarView';
 import { SchemaViewProps } from './types';
 
+// Break circular imports
+const ArrayView = React.lazy(() =>
+  import('./ArrayView').then(m => ({ default: m.ArrayView })),
+);
+const MatchView = React.lazy(() =>
+  import('./MatchView').then(m => ({ default: m.MatchView })),
+);
+const ObjectView = React.lazy(() =>
+  import('./ObjectView').then(m => ({ default: m.ObjectView })),
+);
+const ScalarView = React.lazy(() =>
+  import('./ScalarView').then(m => ({ default: m.ScalarView })),
+);
```

Comment:
so these are a bit strange, how did you arrive at this? 😁

Reply:
I'm not sure! Haha :)

Like, these are truly circularly importing each other and the build warns about that. I just wanted to fix it somehow. Is this approach a problem?

Reply:
perhaps try to merge the recursive components in a single file?

Reply:
Sure. But if not merging all that into one file, I'd still be curious if the lazy thing is unexpected or doesn't work well 🤔